coordinator/tests/services/translation/reversibility.spec.ts

```diff
@@ -0,0 +1,51 @@
+import {convertParentPrescription} from "../../../src/services/translation/request/prescription/parent-prescription"
+import {createInnerBundle} from "../../../src/services/translation/response/release/release-response"
+import * as TestResources from "../../resources/test-resources"
+import {getResourcesOfType} from "../../../src/services/translation/common/getResourcesOfType"
+import * as fhir from "../../../src/models/fhir"
+import * as hl7V3 from "../../../src/models/hl7-v3"
+import * as LosslessJson from "lossless-json"
+import * as uuid from "uuid"
+import {toArray} from "../../../src/services/translation/common"
+
+describe("translations are reversible", () => {
+  const bundle = TestResources.specification[0].fhirMessageSigned
+  const parentPrescription = convertParentPrescription(bundle)
+  ensureLineItemStatus(parentPrescription)
+  const originalBundle = JSON.parse(LosslessJson.stringify(bundle))
+  const translatedBundle = createInnerBundle(parentPrescription, uuid.v4())
+
+  test.skip.each([
```

Comment:
any updates on the skip? are we keeping this in for later?

Reply:
Yeah I'm going to address the test failures in a new PR :+1: